doc/radosgw/cloud-restore.rst

```diff
@@ -226,8 +226,10 @@ Example 3:
 
 This will restore the object ``doc3.rtf`` for ``read_through_restore_days`` days.
 
-.. note:: The above CLI command may time out if object restoration takes too long.
-          You can verify the restore status before reissuing the command.
+The ``rgw_read_through_timeout_ms`` configuration option controls how long the
+GET request will wait for restore completion before returning a timeout error.
+The default is 10000 milliseconds (10 seconds). Set to 0 to disable waiting
+(clients must poll by retrying the GET request).
```

Comment:
Suggest a rephrasing without the parenthetical.  Took me three reads to get what you’re saying here.